hack/setup-management/main.go

```diff
@@ -0,0 +1,303 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/network/mgmt/network"
+	azresources "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/resources"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/sirupsen/logrus"
+
+	armconst "github.com/openshift/openshift-azure/pkg/arm/const"
+	fakerparm "github.com/openshift/openshift-azure/pkg/fakerp/arm"
+	farmconst "github.com/openshift/openshift-azure/pkg/fakerp/arm/const"
+	"github.com/openshift/openshift-azure/pkg/util/arm"
+	"github.com/openshift/openshift-azure/pkg/util/azureclient"
+	"github.com/openshift/openshift-azure/pkg/util/azureclient/resources"
+	"github.com/openshift/openshift-azure/pkg/util/resourceid"
+)
+
+type cidrName string
+
+const (
+	cidrVnet            cidrName = "cidrVnet"
+	cidrDefaultSubnet   cidrName = "cidrDefaultSubnet"
+	cidrGatewaySubnet   cidrName = "cidrGatewaySubnet"
+	cidrManagmentSubnet cidrName = "cidrManagmentSubnet"
+	cidrClients         cidrName = "cidrClients"
+)
+
+var (
+	versionMap = map[string]string{
+		"Microsoft.Network": "2019-02-01",
```

Comment:
RP side is using 2019-04-01 SDK/Network for private link package, RP side is using 2019-04-01 SDK/Network. Is it necessary to sync this on both sides?

Reply:
I can do that just so we are consistent :)